word_language_model/generate.py

```diff
@@ -52,7 +53,29 @@
     parser.error("--temperature has to be greater or equal 1e-3.")
 
 with open(args.checkpoint, 'rb') as f:
-    model = torch.load(f, map_location=device)
+    safe_globals = [
+        PositionalEncoding,
+        TransformerModel,
+        torch.nn.functional.relu,
+        torch.nn.modules.activation.MultiheadAttention,
+        torch.nn.modules.container.ModuleList,
+        torch.nn.modules.dropout.Dropout,
+        torch.nn.modules.linear.Linear,
+        torch.nn.modules.linear.NonDynamicallyQuantizableLinear,
+        torch.nn.modules.normalization.LayerNorm,
+        torch.nn.modules.sparse.Embedding,
+        torch.nn.modules.transformer.TransformerEncoder,
+        torch.nn.modules.transformer.TransformerEncoderLayer,
+        RNNModel,
+        torch.nn.modules.dropout.Dropout,
+        torch.nn.modules.linear.Linear,
+        torch.nn.modules.rnn.GRU,
+        torch.nn.modules.rnn.LSTM,
+        torch.nn.modules.rnn.RNN,
+        torch.nn.modules.sparse.Embedding,
```

Comment:
remove duplicates, use abc order:
```suggestion
        PositionalEncoding,
        RNNModel,
        TransformerModel,
        torch.nn.functional.relu,
        torch.nn.modules.activation.MultiheadAttention,
        torch.nn.modules.container.ModuleList,
        torch.nn.modules.dropout.Dropout,
        torch.nn.modules.linear.Linear,
        torch.nn.modules.linear.NonDynamicallyQuantizableLinear,
        torch.nn.modules.normalization.LayerNorm,
        torch.nn.modules.sparse.Embedding,
        torch.nn.modules.rnn.GRU,
        torch.nn.modules.rnn.LSTM,
        torch.nn.modules.rnn.RNN,
        torch.nn.modules.transformer.TransformerEncoder,
        torch.nn.modules.transformer.TransformerEncoderLayer,
```